tools/CMakeLists.txt

```
@@ -28,3 +28,31 @@ target_link_libraries(sunshinesvc
         wtsapi32
         ${PLATFORM_LIBRARIES})
 target_compile_options(sunshinesvc PRIVATE ${SUNSHINE_COMPILE_OPTIONS})
+
+# Add sunshine_wgc_capture executable
+add_executable(sunshine_wgc_capture
+        sunshine_wgc_capture.cpp
+        ${CMAKE_SOURCE_DIR}/src/platform/windows/ipc/pipes.cpp
+        ${CMAKE_SOURCE_DIR}/src/platform/windows/ipc/misc_utils.cpp
+        ${CMAKE_SOURCE_DIR}/src/logging.cpp)
+set_target_properties(sunshine_wgc_capture PROPERTIES CXX_STANDARD 20)
+
+# Define helper build flag to conditionally compile logging
+target_compile_definitions(sunshine_wgc_capture PRIVATE
+        SUNSHINE_EXTERNAL_PROCESS=1
+)
+
+target_link_libraries(sunshine_wgc_capture
+        ${Boost_LIBRARIES}
+        ${CMAKE_THREAD_LIBS_INIT}
+        d3d11
+        dxgi
+        ole32
+        user32
+        gdi32
+        windowsapp
+        avrt
+        shcore
+        wtsapi32
+        ${PLATFORM_LIBRARIES})
+target_compile_options(sunshine_wgc_capture PRIVATE ${SUNSHINE_COMPILE_OPTIONS})
```

```suggestion
target_compile_options(sunshine_wgc_capture PRIVATE ${SUNSHINE_COMPILE_OPTIONS})

```

File should have an empty blank line at the end. You can configure vs code to do this for you automatically.